deployments/basex-validator/automations/updateautomation.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImageUpdateAutomation
+metadata:
+  name: basex-validator
+  namespace: basex-validator
+spec:
+  interval: 20m
```

Comment:
Think the previous interval was 5 minutes, so maybe something that can be tuned in the future if decreasing the value doesn't impact the pull rate limit from docker hub nor increase system load too much.

Reply:
Thanks. I'll be honest I hadn't thought much about that value. 5 minutes should be fine, it's not going to impact the docker pull rate, it's the work that flux will have to do in cluster. 5 minutes seems reasonable to me - I'd be tempted to move this one more frequently later, depending on what impact I can see in the cluster.